release_notes.md

```diff
@@ -2,10 +2,6 @@
 
 <!-- Please add your release notes in the following format:
 - My change description (#PR)
--->
```

Comment:
Please include this line - it is part of the comment above it. Thank you!

Reply:
Updated @VpOfEngineering